IPs/RSKIP180.md

```diff
@@ -0,0 +1,61 @@
+# Limit the RSK merged mining merkle proof
+
+
+|RSKIP          | 180 |
```

Comment:
replace 180 -> 258 (PR number as placeholder for rskip number)

Looks like a new RSKIP, with no relation to old PR 180

Reply:
This is already specified as RSKIP 180 in Iris